### DISCUSSION TITLE
How to restrict application to local network IPs only?

### DISCUSSION THREAD

**Discussion by circuitikzuser** [netblue30/firejail#5673] [Q&A]
Is it possible to firejail an application to allow network access for the firejailed application ONLY to and from a specified network address ?

At the moment I can completely cut off network access to a firejailed application, but there seems to be no way in firejail to allow access only from the local Lan network or from specified IP addresses.

Anyone know how to od this.

**Comment by rusty-snake** [comment]
If you want to restrict the from, you can use `net IFACE` to restrict the used interface.

For the to you can use `netfilter`.

**Comment by circuitikzuser** [comment]
netfilter cannot do per application blocking, and will block all applications to connect to specific ip's.
If so, then it will not work. 
Or am I wrong about netfilter ?

**Reply by rusty-snake** [reply]
> **netfilter filename**
> If a new network namespace is created, enabled the network filter in filename.

Why should this affect applications run in a different profile than you configured it?
